checkov/common/parallelizer/parallel_runner.py

```diff
@@ -33,10 +33,13 @@ def __init__(
             # PYCHARM_HOSTED env variable equals 1 when debugging via jetbrains IDE.
             # To prevent JetBrains IDE from crashing on debug run sequentially
             self.type = ParallelizationType.NONE
-        elif self.os == "Windows":
-            # 'fork' mode is not supported on 'Windows'
-            # 'spawn' mode results in a strange error, which needs to be investigated on an actual Windows machine
-            self.type = ParallelizationType.THREAD
+        elif self.os == "Windows" or self.os == "Darwin":
```

Comment:
can you also allow to overwrite the behaviour via the `CHECKOV_PARALLELIZATION_TYPE` env var here? Maybe do an initial if condition check for the env var and use it instead of overwriting it for both OS, then users could also set it to none, if needed?

Reply:
Sure I'll add